source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -1108,6 +1111,27 @@ namespace
         }
     }
 
+    void readUnitDefinitions(DocumentPtr doc)
```

Comment:
These are the main utilities to cache unit information on library read, and write unit information for validation.